aws/lambdas/api_end_to_end_test.tf

```diff
@@ -0,0 +1,53 @@
+#
+# API end to end test
+#
+
+resource "aws_lambda_function" "api_end_to_end_test" {
+  function_name = "api-end-to-end-test"
+  image_uri     = "${var.ecr_repository_lambda_urls["api-end-to-end-test-lambda"]}:latest"
+  package_type  = "Image"
+  role          = aws_iam_role.lambda.arn
+  timeout       = 300
```

Comment:
Is there any risk that the lambda times out before the test is completed?

Reply:
With the number of tests we run in this first version of the API end to end test, 5 minutes should be plenty of time. At least with my local testing it was so I will be monitoring once it is up on Staging and tweak as needed.